src/data/fintype/intervals.lean

```diff
@@ -35,19 +35,41 @@ instance Ico_ℤ_fintype (l u : ℤ) : fintype (Ico l u) :=
 fintype.of_finset (finset.Ico_ℤ l u) $
   (λ n, by { simp only [mem_Ico, finset.Ico_ℤ.mem], })
 
+instance Ioo_ℤ_fintype (l u : ℤ) : fintype (Ioo l u) :=
+fintype.of_finset (finset.Ico_ℤ (l+1) u) $ λ n,
+  by simp only [mem_Ioo, finset.Ico_ℤ.mem, int.add_one_le_iff, iff_self, implies_true_iff]
```

Comment:
```suggestion
instance Ioo_ℤ_fintype (l u : ℤ) : fintype (Ioo l u) := set.Ico_ℤ_fintype (l + 1) u
```

Reply:
Didn't work when I tried it

Reply:
I just tested.

Reply:
Ooh, well, then I would rather keep it the way it is. It seems a little bit fragile if it works in some cases but not in other cases.